src/Persistence.LinnApps/ServiceDbContext.cs

```diff
@@ -509,6 +512,17 @@ private void BuildPurchaseOrderDeliveries(ModelBuilder builder)
             entity.Property(o => o.NetTotal).HasColumnName("NET_TOTAL").HasMaxLength(18);
         }
 
+        private void BuildOverbookAllowedBy(ModelBuilder builder)
+        {
+            var entity = builder.Entity<OverbookAllowedByLog>().ToTable("PL_OVERBOOK_ALLOWED_BY");
+            entity.HasKey(a => a.Id);
```

Comment:
you haven't actually told the mapper what the Id fields column name is, and you have two keys  on the OverbookAllowedByLog, Id and LogId.  I would guess that's probably why the insert isn't working